static/locales/de-DE.yaml

```diff
@@ -97,7 +97,7 @@ User Playlists:
 History:
   # On History Page
   History: Verlauf
-  Watch History: Schaue Verlauf
```

Comment:
Here it depends on whether this is for a button which starts playing all the videos in the history or whether it is a button to say "Take a look at your history". I assumed the first one.

Reply:
I think "Verlauf anschauen" sounds better and can be understood ambiguously

Reply:
thx for your help :smile: